packages/dashboard/src/components/sidePanel/sections/baseSettingSection/utils.ts

```diff
@@ -0,0 +1,15 @@
+/**
+ * Widgets can only be 2x2 to 100x100. This function checks if the value is within that range.
+ */
+export function isValidWidgetDimension(value: number) {
+  return value >= 2 && value <= 100;
```

Comment:
Blocker: Remove function and instead rely on [`<Controller rules={..} />`](https://react-hook-form.com/api/usecontroller/controller/) for validation.